### PR TITLE
Bump minor versions of wdio, and vite to v2.X

### PR DESCRIPTION
**Pull request by brickfungus**
* Updated all minor versions
* Updated vite to v2 and made necessary updates to coverage file
* removed unnecessary awaits in index test file

### CODE REVIEW

**Comment by brickfungus** [comment]
I left eslint to v 8.X to match rest of wdio

File: src/index.test.ts [hunk 1]
```diff
@@ -181,30 +181,30 @@ describe('Video Reporter', () => {
     it('should add frame if no alert is displayed', async () => {
```
Comment (christian-bromann) [inline src/index.test.ts:181]: Can we remove all `async` in tests where we don't use `await`?

**Review by christian-bromann — CHANGES_REQUESTED** [review]
One minor request

File: src/index.test.ts (hunk 1 above)
Comment (brickfungus) [reply src/index.test.ts]: I think i got them all

**Review by christian-bromann — APPROVED** [review]
Awesome, lgtm 👍

**Pull request merged**